packages/storybook/stories/Examples/TeamsInterop/snippets/CallComponent.snippet.tsx

```diff
@@ -1,6 +1,6 @@
 import { StreamMedia, VideoTile } from '@azure/communication-react';
 import React from 'react';
-import { renderVideoStream } from '../../../utils';
+import { useVideoStream } from '../../../utils';
```

Comment:
If a snippet file references utils like this then in the snippet it will show as `import { useVideoStream } from '../../../utils';` which won't work when copy-pasted. This doesn't need fixed in this PR but this it may get more complex to fix if we don't schedule soon to fix this.

Reply:
Good point, though I don't really have a great solution for this (it'll probably just amount to duplicating some of the code to the snippet) but I've added this bug for tracking. https://skype.visualstudio.com/SPOOL/_workitems/edit/2460682. I can try to tackle this in a separate PR.